scripts/utility.py

```diff
@@ -301,13 +301,14 @@ def change_clan_relations(other_clan, difference):
 
 
 def create_new_cat_block(
```

Comment:
Why can't you import `CatAgeEnum` into `utility.py` directly?

Reply:
`scripts.cat.cats` imports `scripts.utility` so doing so would cause a "partially initialized module" error due to the circular import